src/DynamoCoreWpf/Views/CodeCompletion/CodeCompletionEditor.xaml.cs

```diff
@@ -50,8 +51,9 @@ public CodeCompletionEditor(NodeView nodeView)
 
             nodeView.Unloaded += (obj, args) => IsDisposed = true;
             this.nodeViewModel = nodeView.ViewModel;
-            this.dynamoViewModel = nodeViewModel.DynamoViewModel;
             this.DataContext = nodeViewModel.NodeModel;
+            this.dynamoViewModel = nodeViewModel.DynamoViewModel;
+            this.dynamoViewModel.PropertyChanged += OnDynamoViewModelPropertyChanged;
```

Comment:
do you think that it is worth adding a new recordableCommand for this setting? I think this is how some of the other settings work?

if not we should unsubscribe this I think when the code block view is closed?

Reply:
This is what I mentioned as TODO - Disposal guard